pyscriptjs/src/pyodide.ts

```diff
@@ -89,8 +92,38 @@ export class PyodideRuntime extends Runtime {
         if (package_name.length > 0) {
             logger.info(`micropip install ${package_name.toString()}`);
             const micropip = this.globals.get('micropip') as Micropip;
-            await micropip.install(package_name);
-            micropip.destroy();
+            try {
+                await micropip.install(package_name);
+                micropip.destroy();
+            } catch(e) {
```

Comment:
here we should check that `e` is actually a Python error and not some unrelated exception that we don't expect

Reply:
In that case we are adding the exception message to the banner. Would that be enough you think?
If we handle only `PythonError` exceptions, the others will only be visible in the console logs and the splashscreen wont be closed 🤔

Reply:
> In that case we are adding the exception message to the banner. Would that be enough you think? 

I suppose we can live with that, but it's conceptually wrong, see below.

> If we handle only `PythonError` exceptions, the others will only be visible in the console logs and the splashscreen wont be closed thinking

yes, but this is a separate problem: generally speaking, Python errors are caused by the user, but JS errors are caused by us: ideally, we should catch them and display a banner to the user which says "pyscript internal error, please open an issue": this way, they know that it's not *their* fault.